Source/WarpX.cpp

```diff
@@ -272,6 +274,10 @@ WarpX::ReadParameters ()
 
     ReadBoostedFrameParameters(gamma_boost, beta_boost, boost_direction);
 
+    do_compute_max_step_from_zmax = 
+        pp.query("zmax_plasma_to_compute_max_step", 
+                  zmax_plasma_to_compute_max_step);
```

Comment:
Could you add a quick comment explaining that `pp.query` returns 1 if  `zmax_plasma_to_compute_max_step` is in the input deck?

Reply:
Sure, this was included in the latest commit